jsoc/gsoc/generativeai.md

```diff
@@ -0,0 +1,107 @@
+# JuliaGenerativeAI Projects
+
+JuliaGenerativeAI is an organization focused on advancing Generative AI research and looking for its applications within the Julia programming language ecosystem. Our community comprises AI researchers, developers, and enthusiasts passionate about pushing the boundaries of Generative AI using Julia's high-performance capabilities. We strive to create innovative tools and solutions that leverage the unique strengths of Julia in handling complex AI challenges.
+
+There is a high overlap with organizations, you might be also interested in:
+- [Projects with MLJ.jl](https://julialang.org/jsoc/gsoc/MLJ/) - For more traditional machine learning projects
+- [Projects in Reinforcement Learning](https://julialang.org/jsoc/gsoc/machine-learning/) - For projects around AlphaZero.jl
+- [Projects with FluxML](https://fluxml.ai/gsoc/) - For projects around Flux.jl, the backbone of Julia's deep learning ecosystem
+
+## Large Language Model Projects
+
+### Project 1: Enhancing llama2.jl with GPU Support
+
+**Description:** llama2.jl is a Julia native port for Llama architectures, originally based on [llama2.c](https://github.com/karpathy/llama2.c). This project aims to enhance llama2.jl by implementing GPU support through KernelAbstraction.jl, significantly improving its performance.
```

Comment:
Which one do we want to point people to?

https://github.com/cafaxo/Llama2.jl

or 

https://github.com/juvi21/llama2.jl

?

Reply:
I'm familiar with cafaxo's and it has >100 stars. I don't think the other one has GGUF support and other important bits. Sixzero was definitely referring to cafaxo's repo

Reply:
```suggestion
**Description:** [Llama2.jl](https://github.com/cafaxo/Llama2.jl) is a Julia native port for Llama architectures, originally based on [llama2.c](https://github.com/karpathy/llama2.c). This project aims to enhance Llama2.jl by implementing GPU support through KernelAbstraction.jl, significantly improving its performance.
```